lib/project_types/extension/cli.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+require 'pathname'
+require 'json'
+require 'yaml'
```

Comment:
We're knowingly compromising on load time here in favour of robustness. 20ms in additional load time is an acceptable trade off for not running into production issues due to missing dependencies. Given that Ruby's require is global its way too easy to be fooled by our test suite and think that everything is fine when reality that very much depends in which order files are loaded in production.